proto/src/determined/api/v1/experiment.proto

```diff
@@ -20,12 +20,16 @@ import "determined/experiment/v1/searcher.proto";
 // One datapoint in a series of metrics from a trial in batch.
 message DataPoint {
   option (grpc.gateway.protoc_gen_swagger.options.openapiv2_schema) = {
-    json_schema: { required: [ "batches", "value" ] }
+    json_schema: { required: [ "batches", "value", "time" ] }
   };
   // Total batches processed by the time this measurement is taken.
   int32 batches = 1;
   // Value of the requested metric at this point in the trial.
   double value = 2;
+  // The time the measurement is taken.
+  google.protobuf.Timestamp time = 3;
+  // The epoch this measurement is taken.
+  optional int32 epoch = 4;
 }
 
 // One DataPointTime in a series of metrics from a trial in time.
```

Comment:
suggestion: remove these?

Reply:
I only kept them because I thought we might need/want to support some type of backwards compatibility but I agree.